backend/api/Services/RobotService.cs

```diff
@@ -28,5 +40,32 @@ public async Task<IEnumerable<Robot>> ReadAll()
         {
             return await _context.Robots.FirstOrDefaultAsync(robot => robot.Id.Equals(id, StringComparison.Ordinal));
         }
+
+        public async Task<Report> StartMission(Robot robot, string missionId)
+        {
+            var response = await _isarService.StartMission(missionId);
+            if (!response.IsSuccessStatusCode)
+            {
+                throw new MissionException($"Could not start mission with id: {missionId}");
+            }
+            if (response.Content != null)
```

Comment:
```suggestion
            if (response.Content is null) throw new MissionException("Could not read content from mission");
```
I prefer to check for `null` rather than `!null` so we don't have end of if statements at the bottom which could have finished earlier. Easier to read too.

Reply:
Also, do we need to check that the content is not null once we have checked for status code 200? There will be a lot of null checks but might be it's required.

Reply:
Fixed